src/Components/Article.jsx

```diff
@@ -129,6 +131,26 @@ function Article(){
         setNewComment("")
     }
 
+    if (invalidArticle !== "none") {
```

Comment:
setting the original state to be false and then just checking if it's truthy here would be cleaner